test/ovn/OVN_Northbound.dl.expected

```diff
@@ -1,132 +1,69 @@
 import types
 
-input relation NB_Global(
+input relation NB_Global (
     _uuid: uuid,
     nb_cfg: integer,
     sb_cfg: integer,
     hv_cfg: integer,
+    external_ids: Map<string,string>,
```

Comment:
I am wondering whether this will work for nested maps: `Map<string, Map<string, string>>`.